src/firebaseConfig.js

```diff
@@ -0,0 +1,10 @@
+// import { env } from '$lib/env'
+
+export default {
+  apiKey: import.meta.env.VITE_FIREBASE_API_KEY,
```

Comment:
@billdevmaster it should be safe to load the API key from a config file during the build process, so that the site remains a compatible with a static one and deployment to GitHub Pages

See https://firebase.google.com/docs/projects/api-keys#api-keys-for-firebase-are-different

Reply:
yes. so I added .env file to project.
Do you want the .env content now?

Reply:
Yup that way it's all testable, I tried to run the app locally and it failed with an error about the missing key

Reply:
please create the .env file in root path of project.
the content is 
VITE_FIREBASE_API_KEY=AIzaSyD1f5oGdXLaI6ZGJ4RIouvOM3_RHm0qJFs

Reply:
Is this working?